handbook/product/planning.md

```diff
@@ -0,0 +1,50 @@
+# Planning
+
+Planning is a continuous process of negotiation between product and engineering. Teams should review the project roadmap weekly with the [Product Manager](roles.md#product-manager) during their team meetings, and update it regularly as new information comes in. The product manager should also review the plans for all projects in the next release for overall coherency. This ensures that projects can work on the schedules that make the most sense for them (subject to the constraint of needing to ship some kind of milestone monthly). A particularly long-term project can have many months of visibility into requirements and plans, and shorter or more experimental projects can be planned with shorter time horizons.
+
+## Planning artifacts
+
+Planning requires several artifacts for communicating. This section clarifies how each artifact is used in the planning process.
+
+### OKRs
+
+[OKRs](../../okrs/index.md) align teams with the company's goals for the quarter. They should be used to make prioritization decisions and help to maintain team direction and focus.
+
+### Project roadmap
+
+[The project roadmap](https://docs.google.com/document/d/1cBsE9801DcBF9chZyMnxRdolqM_1c2pPyGQz15QAvYI/edit?usp=sharing) is where we track and prioritize larger projects that we intend to finish in the next 3-6 months.
+
+Engineering should feel empowered to add items to the roadmap that they feel strongly about, which will start conversations with product. We want to push decisions down to people closest to those problems. It is product's responsibility to help give insight into customer pains and feedback, strategic priorities, and to ensure consistency across the product.
+
+The project roadmap is NOT:
+
+- a comprehensive backlog of everything we want to do
+- a list of every bug or customer issue in our issue tracker
+- permanent or mandated
+
+The project roadmap is:
+
+- a prioritized list of the most important things we intend to do
+- an indicator of relative priority to teams, customers, and users
+- a place for discussions about priority
+- where engineering and product negotiate the next most important thing to work on
+
+Teams should work on the next thing at the top of the roadmap and should not jump around. If there is a desire to work on something lower on the roadmap, that should be elevated during team meetings and the chnage in priority should be discussed. That item should be moved in the roadmap based on the outcome of the discussion.
+
+### RFCs
+
+[RFCs](../communication/rfcs/index.md) are ephemeral documents used to outline and solicit feedback on projects. They provide context around a problem that needs to be solved and are a medium for creating specifications for a project.
```

Comment:
I think ephemeral is too strong of a word here. RFCs don't go away.

Reply:
Good point! Updating in follow up.